plugins/00_dokku-standard/subcommands/deploy

```diff
@@ -9,7 +9,7 @@ dokku_deploy_cmd() {
   declare desc="deploy phase"
   local cmd="deploy"
   [[ -z $2 ]] && dokku_log_fail "Please specify an app to deploy"
-  local APP="$2"; local IMAGE_TAG="$3"; local IMAGE=$(get_app_image_name "$APP" "$IMAGE_TAG")
+  local APP="$2"; local IMAGE_TAG="$3"; local IMAGE=$(get_deployed_app_image_name "$APP" "$IMAGE_TAG")
```

Comment:
i think this is a misnomer. reading that function name makes me think it will return the image name from an app that is already deployed. here we want to know the image name that you want to deploy, right? 

something like `get_deploying_app_image_name()` perhaps?